src/components/footer/footer.jsx

```diff
@@ -1,8 +1,77 @@
 import styles from "@/styles/components/footer/footer.module.scss";
 
+import {useTranslations} from 'next-intl';
+import Image from "next/image";
+import Link from "next/link";
+
+import React, { Component } from "react";
+import { AiOutlineInstagram } from "react-icons/ai";
+import { MdOutlineMailOutline, MdFacebook } from "react-icons/md";
+import { HiOutlinePhone } from "react-icons/hi";
+
 export default function Footer() {
+
+  const t = useTranslations('Footer');
+
   return (
-    <div> Footer yay</div>
+    <footer className={styles.footer}>
+        <div className={styles.body}>
+            {/* nav buttons */}
+            <ul className={styles.pages}>
+                <li>
+                    <Link className={styles.link} href="/">{t('pages.page1')}</Link>
+                </li>
+                <li>
+                    <Link className={styles.link} href="/">{t('pages.page2')}</Link>
+                </li>
+                <li>
+                    <Link className={styles.link}  href="/">{t('pages.page3')}</Link>
+                </li>
+                <li>
+                    <Link className={styles.link}  href="/">{t('pages.page4')}</Link>
+                </li>
+                <li>
+                    <Link className={styles.link} href="/">{t('pages.page5')}</Link>
+                </li>
+            </ul>
+        
+            {/* contacts */}
+            <div className={styles.contact}> 
+                <h3>{t('contact.message')}</h3>
+                <div className={styles.icons}>
+                    <a href="https://includedavis.com" target="_blank" className={styles.circle_frame_facebook}>
+                        <MdFacebook/> 
+                    </a>
+                    <a href="https://includedavis.com" target="_blank" className={styles.circle_frame}>
+                        <AiOutlineInstagram/>
+                    </a>
+                    <a href="https://includedavis.com" target="_blank" className={styles.circle_frame}>
+                        <HiOutlinePhone/>
+                    </a>
+                    <a href="https://includedavis.com" target="_blank" className={styles.circle_frame}>
+                        <MdOutlineMailOutline/>
+                    </a>
+                </div>
+                <p>{t('contact.address_pt1')}<br/>{t('contact.address_pt2')}</p>
+            </div>
+
+            {/* thank you message */}
+            <div className={styles.sponsor}>
+                <p>{t('sponsor_pt1')}<br/>{t('sponsor_pt2')}</p>
+                <Image
```

Comment:
I'll touch base about this with everyone since it just came to mind, we'll be wrapping every Image component with a div to help with responsive design later on. Just as an FYI